src/components/ConnectWallet/Option.jsx

```diff
@@ -10,7 +10,7 @@ export const Option = (props) => {
           href="https://metamask.io/"
           target="_blank"
           rel="noreferrer noopener"
-          className="inline-flex w-full text-sm mb-5 font-normal text-black-900 bg-white border border-transparent rounded-md focus:border-blue-500 focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-blue-500"
+          className="inline-flex w-full text-sm mb-5 font-normal text-black-900 bg-white border border-[#D4DFEE] rounded-lg focus:border-blue-500 focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-blue-500"
```

Comment:
```suggestion
          className="inline-flex w-full text-sm mb-5 font-normal text-black-900 bg-white border border-d4dfee rounded-lg focus:border-blue-500 focus:outline-none focus-visible:ring-2 focus-visible:ring-offset-2 focus-visible:ring-blue-500"
```

Don't use arbitrary values like this.
Add them to `tailwind.config.js` theme instead

Reply:
done